front-end/src/pages/home.js

```diff
@@ -219,4 +204,38 @@ const RegenerateRequested = (props) => {
   }
 }
 
+const Group = (props) => {
```

Comment:
small tip, look into this js feature called `destructuring`. 
You can basically use this to extract keys out of your object. Instead of doing 

**_this_**
```
let group = props.group
let classes = props.classes
let handleVisit = props.handleVisit
```
You could just 

```let {group, classes, handleVisit} = props```

which would extract the keys with those same variable names from `props` object and create a local copy with those same variable names.

You can directly destructure your keys from within your function argument as well, by doing something like this

```const Group = ({group, classes, handleVisit}) => {..}```

Might want to try it out

Reply:
Nothing wrong with keeping your stuff as is as well, just something you might want to explore